trendico/migrations/0004_product_total_quantity_topsellingproduct.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.3 on 2023-08-03 06:06
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('trendico', '0003_remove_product_label'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='product',
+            name='total_quantity',
+            field=models.PositiveIntegerField(default=10),
+            preserve_default=False,
+        ),
+        migrations.CreateModel(
+            name='TopSellingProduct',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('total_sales_quantity', models.PositiveIntegerField()),
+                ('total_revenue', models.DecimalField(decimal_places=2, max_digits=10)),
+                ('profit_margin', models.DecimalField(decimal_places=2, max_digits=5)),
+                ('customer_ratings', models.DecimalField(decimal_places=2, max_digits=3)),
+                ('promotional_success', models.BooleanField()),
+                ('market_trends', models.BooleanField()),
+                ('last_updated', models.DateTimeField(auto_now=True)),
+                ('product', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='trendico.product')),
+            ],
+        ),
+    ]
```

Comment:
There should be one migration to have all these changes, multiple migrations for one model changes are not recommended